zetaclient/bitcoin/bitcoin_client_live_test.go

```diff
@@ -348,3 +348,76 @@ func LiveTestAvgFeeRateTestnetMempoolSpace(t *testing.T) {
 
 	compareAvgFeeRate(t, client, startBlock, endBlock, true)
 }
+
+// Remove prefix "Live" to run this live test
```

Comment:
Can we add some comments (here and other functions) what the test is about?

Reply:
https://github.com/zeta-chain/node/pull/1962/commits/a61d8886c7bda94ed7b6d8adba74ed6809aa575a